### DISCUSSION TITLE
when phone number based authentication can be expected?

### DISCUSSION THREAD

**Discussion by iamdineshkumar** [appwrite/appwrite#1467] [Q&A] [answered]
Whether this feature will be done by having account with service providers like twilio in that case which service providers supported and is that possible from our end to add other service providers through creating adaptors/plugin?

**Comment by TorstenDittmann** [comment]
We have this on our roadmap and will be creating a RFC for this in the near future, but we don't have date for this yet.

**Comment by crapthings** [comment]
maybe we can support username or mobile number easily by toggling the login field

**Comment by stnguyen90** [comment, marked as answer]
As of [Version 0.15.0](https://github.com/appwrite/appwrite/releases/tag/0.15.0), Appwrite supports phone authentication. We welcome PRs for additional providers.
